src/request.ts

```diff
@@ -452,19 +445,26 @@ class DatastoreRequest {
    *   const entities = data[0];
    * });
    */
-  get(keys, options?): Promise<EntityDataObj[]>;
-  get(keys, options?, callback?): void|Promise<EntityDataObj[]> {
-    if (is.fn(options)) {
-      callback = options;
-      options = {};
-    }
-
-    options = options || {};
+  get(keys: Entities, options?: CreateReadStreamOptions): Promise<Entity|Stream>;
+  get(keys: Entities, callback: GetCallback): void;
+  get(keys: Entities, options: CreateReadStreamOptions, callback: GetCallback): void;
+  get(keys: Entities, optionsOrCallback?: CreateReadStreamOptions|GetCallback, cb?: GetCallback): void|Promise<Entity|Stream> {
+    if (typeof optionsOrCallback === 'function' && typeof optionsOrCallback !== 'object') {
```

Comment:
Well I mean both of these can't really be true? 
```js
const options = typeof optionsOrCallback === 'object' ? optionsOrCallback : {};
const callback = typeof optionsOrCallback === 'function' ? optionsOrCallback : cb;
```

Reply:
Nope! But the linter likes it 😆 Fixed!